public/posts/v1.6.md

```diff
@@ -0,0 +1,226 @@
+Today we are releasing Deno 1.6.0. This release contains some major features,
+and many bug fixes. Here are some highlights:
+
+- [**Build self contained, standalone binaries**](#codedeno-compilecode-building-self-contained-standalone-binaries):
+  `deno compile` can build your Deno projects into completly standalone
+  executables
+- [**Built-in Deno Language Server**](#built-in-deno-language-server): fully
+  integrated LSP for editor integration
+- [**Experimental aarch64-apple-darwin support**](#arm64): release binaries for
+  Apple's new M1 Arm64 chip.
+
+If you already have Deno installed you can upgrade to 1.6 by running
+`deno upgrade`. If you are installing Deno for the first time, you can use one
+of the methods listed below:
+
+```shell
+# Using Shell (macOS and Linux):
+curl -fsSL https://deno.land/x/install/install.sh | sh
+
+# Using PowerShell (Windows):
+iwr https://deno.land/x/install/install.ps1 -useb | iex
+
+# Using Homebrew (macOS):
+brew install deno
+
+# Using Scoop (Windows):
+scoop install deno
+
+# Using Chocolatey (Windows):
+choco install deno
+```
+
+# New features and changes
+
+## `deno compile`: building self-contained, standalone binaries
+
+<img src="/posts/v1.6/compile.gif">
+
+Ever since Deno's inception, we have aimed to provide developer utilities, that
+are often external programs in other ecosystems, in the Deno CLI out of the box.
+Examples of this are `deno fmt`, and `deno lint`. Today we are pleased to add
+another developer tool to the Deno toolchain: `deno compile`.
+
+`deno compile` does for Deno what `nexe` or `pkg` do for Node: create a
+standalone, self-contained binary from your JavaScript or TypeScript source
+code. This has been the single most upvoted issue on the Deno issue tracker, and
+we are glad to be able to deliver a first pass for this feature.
+
+You can try it out yourself now:
+
+```shell
+$ deno compile --unstable https://deno.land/std@0.79.0/http/file_server.ts
+Check https://deno.land/std@0.79.0/http/file_server.ts
+Bundle https://deno.land/std@0.79.0/http/file_server.ts
+Compile https://deno.land/std@0.79.0/http/file_server.ts
+Emit file_server
+
+$ ./file_server
+HTTP server listening on http://0.0.0.0:4507/
+```
+
+The `deno compile` subcommand is currently still unstable, because we want to
+investigate what use-cases you as a community can come up with, that might not
+yet be optimally covered. If you have feedback, please comment on
+[Deno discord](https://discord.gg/deno), or create an issue with feature
+requests on the [Deno issue tracker](https://github.com/denoland/deno/issues).
```

Comment:
Should we add short description how it works under the hood? I'm sure there's gonna be a lot of question about it